runtime/src/main/java/io/quarkiverse/logging/cloudwatch/LoggingCloudWatchHandler.java

```diff
@@ -84,11 +85,12 @@ private boolean isBelowThreshold(LogRecord record) {
 
     @Override
     public void flush() {
-        done = true;
     }
 
     @Override
     public void close() throws SecurityException {
+        LOGGER.info("Shutting down and awaiting termination");
+        shutdownAndAwaitTermination(scheduler);
```

Comment:
After the shutdown is complete, `eventBuffer` can still contain log entries (up to 5 seconds) that were not pushed and will be lost.

We should do a last call to `awsLogs.putLogEvents` here to prevent data loss.